docs/development/maintainer-info.rst

```diff
@@ -36,6 +36,12 @@ First install `ghp-import <https://github.com/davisp/ghp-import>`__
 
     pip install ghp-import
 
+Run ``towncrier`` to update the changelog (set a version number if this is a release):
```

Comment:
This needs to be in the `how to make a release` section, since this should only happen right before the next release.